cis194/week1/daniel-mc/Homework.hs

```diff
@@ -0,0 +1,50 @@
+module Homework
+  ( toDigits
+  , doubleEveryOther
+  , sumDigits
+  , validate
+  , hanoi
+  , hanoi4
+  )
+where
+
+-- Exercise 1
+toDigitsRev :: Integer -> [Integer]
+toDigitsRev x | x <= 0    = []
+              | otherwise = (x `mod` 10) : toDigitsRev (x `div` 10)
+toDigits :: Integer -> [Integer]
+toDigits = reverse . toDigitsRev
+
+-- Exercise 2
+
+doubleEveryOtherRev :: [Integer] -> [Integer]
+doubleEveryOtherRev []             = []
+doubleEveryOtherRev [x           ] = [x]
+doubleEveryOtherRev (x1 : x2 : xs) = x1 : (x2 * 2) : doubleEveryOtherRev xs
+
+doubleEveryOther :: [Integer] -> [Integer]
+doubleEveryOther = reverse . doubleEveryOtherRev . reverse
```

Comment:
I think @shineli1984 mentioned on my PR as well there's no need to `reverse` back

Reply:
Oh that's true since it's summed anyway, but then the tests for doubleEveryOther would fail

Reply:
@shineli1984 I think that makes sense. Should we merge this?

Reply:
I think Daniel can merge it when he's happy with it. But it LGTM.

Reply:
Thanks yeah I'm happy with it. I don't think I have permission to merge